CONTRIBUTING.md

```diff
@@ -167,13 +180,17 @@ any PMC member:
 1. Merge the release proposal PR into the master branch of the repository. There
    should not be any conflicts as the text in the files that have changed should
    only be changed during this release process.
-1. Create a new branch called **"v$major.$minor"** from the **Release SHA**.
+1. Checkout the **Release SHA** (either to a local branch or in 'detached HEAD'
+   state). This is done in order to ensure that changes that were merged into
+   master after the release candidate was created are not included in the
+   release.
 1. Cherry pick the commit in which the release proposal PR was merged, to pick
    up the file changes.
 1. Create a new commit updating [spec.md](spec.md), [profile](profile.md),
    [openapi.yaml](openapi.yaml) and [swagger.yaml](swagger.yaml) to include the
    version of the release `v$major.$minor`.
-1. Push the branch to the repository (`v$major.$minor`).
+1. Tag the commit with the name `v$major.$minor`.
```

Comment:
Won't the SHA of this commit (which is the real commit for the release) be different from what you mention in the previous set of changes above?  Perhaps what's missing is that "Release SHA" should be "Release RC SHA" ?

Reply:
Yes - that makes sense. I'll update!

Reply:
Updated @duglin